pkgs/applications/science/logic/anylogic/default.nix

```diff
@@ -0,0 +1,67 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  gnutar,
+  steam-run,
+  makeDesktopItem,
+  copyDesktopItems,
+}: let
+  desktopItem = makeDesktopItem {
+    desktopName = "AnyLogic Personal Learning Edition";
+    genericName = "AnyLogic Personal Learning Edition - for beginners and students";
+    categories = ["Science"];
+    exec = "anylogic";
+    name = "anylogic-ple";
+    icon = "anylogic-ple";
+  };
+in
+  stdenv.mkDerivation rec {
+    pname = "anylogic-ple";
+    version = "8.8.2";
+
+    src = fetchurl {
+      url = "https://files.anylogic.com/${pname}-${version}.linux.x86_64.tgz.bin";
+      sha256 = "0n372fq6q1pr0y3qxwdf5xxx9phmlyrjjgji861qilrqdg4cmzcp";
+    };
+
+    nativeBuildInputs = [copyDesktopItems gnutar];
+    buildInputs = [steam-run];
+
+    unpackPhase = ":";
+    installPhase = ''
+      runHook preInstall
+      tail -n +374 $src > anylogic_extracted.tgz
```

Comment:
Using line numbers is brittle. Can we do this better?

Reply:
That is what the official installer does, as well as what is done in aur
https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=anylogic-ple
Any better suggestion?
![image](https://github.com/NixOS/nixpkgs/assets/10415894/8e59bb3d-2af5-4b72-9df8-b4b0ffce0b43)